server/index.js

```diff
@@ -34,6 +35,12 @@ function main() {
     this.set('Timing-Allow-Origin', '*');
     yield next;
   });
+
+
+  app.use(cors({
+    origin: true // do something more specific here
```

Comment:
` // do something more specific here` - what does this mean? Is it copy-n-paste code?